code/modules/projectiles/gun.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun
 	name = "gun"
-	desc = "Its a gun. It's pretty terrible, though."
+	desc = "A gun, that you can bear. SHALL. NOT. BE. INFRINGED."

```

Comment:
Defines, also you're really not supposed to get your hands on this. Maybe desc inheritance if someone doesn't set it down the line